libdevice/include/device-sanitizer-report.hpp

```diff
@@ -0,0 +1,76 @@
+//==-- assert-happened.hpp - Structure and declaration for assert support --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+
+// Treat this header as system one to workaround frontend's restriction
+#pragma clang system_header
+
+#include <cinttypes>
+
+#ifdef __SPIR__
+
+enum class DeviceSanitizerErrorType : int32_t {
+  UNKNOWN,
```

Comment:
suggest to add a prefix for all error type like "SE_"

Reply:
"enum class" forces to use scope "DeviceSanitizerErrorType::" when the code uses enum values, so I don't think it needs to add a special prefix here.